docs/src/components/Fragment.tsx

```diff
@@ -46,9 +46,10 @@ export const Fragment = ({
   platforms,
   useCommonWebContent,
 }: FragmentProps) => {
-  const { query } = useCustomRouter();
-  const framework = (query.platform as string) ?? 'react';
-  const platform = getPlatform(framework, useCommonWebContent);
+  const {
+    query: { platform: framework = 'react' },
+  } = useRouter();
```

Comment:
These lines seem to be repeated a lot, may be worth having a `useReactRouter` hook for it

Reply:
hmmm.... 🤔  do we reduce much code that way? It's basically a one-liner destructure already.

Reply:
That's fair, I was also thinking maybe we'll want to add more in the future. won't block.